superset-frontend/src/components/CopyToClipboard/index.jsx

```diff
@@ -88,15 +88,7 @@ class CopyToClipboard extends React.Component {
 
   renderNotWrapped() {
     return (
-      <Tooltip
-        id="copy-to-clipboard-tooltip"
-        placement="top"
-        style={{ cursor: 'pointer' }}
-        title={this.props.tooltipText}
-        trigger={['hover']}
-      >
-        {this.getDecoratedCopyNode()}
-      </Tooltip>
+      this.getDecoratedCopyNode()
```

Comment:
It was a generic component and it was used in other components. You shouldn't remove the tooltip.

<img width="392" alt="image" src="https://user-images.githubusercontent.com/2016594/154188857-43be3a1a-96c4-4df8-8758-a67c993cd7c5.png">

Reply:
Agreed, the solution should be prop-based. Either
* render the tooltip conditionally (if it was passed a meaningful title)
* provide a prop or other means (e.g. title={false}) to intentionally hide the tooltip